www/src/pages/components/button.mdx

```diff
@@ -21,51 +22,125 @@ import SingleComponentStatus from '../../components/SingleComponentStatus';
   }
 />
 
+<p style={{ fontSize: '1.5em' }}>
+  This is a pass through component from React-Bootstrap.<br/>
+  <a href="https://react-bootstrap.github.io/components/cards/" target="_blank" rel="noopener noreferrer">
+    See React-Bootstrap for documentation.
+  </a>
+</p>
+
+##### Basic Usage
+```jsx live
+<>
+<div className="mb-3">
+  <Button variant="primary">Primary</Button>{' '}
+  <Button variant="secondary">Secondary</Button>{' '}
+  <Button variant="success">Success</Button>{' '}
+  <Button variant="warning">Warning</Button>{' '}
+  <Button variant="danger">Danger</Button>{' '}
+</div>
+<div className="mb-3">
+  <Button variant="outline-primary">Primary</Button>{' '}
+  <Button variant="outline-secondary">Secondary</Button>{' '}
+  <Button variant="outline-success">Success</Button>{' '}
+  <Button variant="outline-warning">Warning</Button>{' '}
+  <Button variant="outline-danger">Danger</Button>{' '}
+</div>
+<div className="mb-3">
+  <Button variant="light">Light</Button> <Button variant="dark">Dark</Button>{' '}
+  <Button variant="outline-light">Light</Button>{' '}
+  <Button variant="outline-dark">Dark</Button>
+  <Button variant="link">Link</Button>
+  <Button variant="link" className="px-0">No padding link</Button>
+</div>
+</>
+```
+
+##### Size
+
+```jsx live
+<>
+  <div className="mb-2">
+    <Button variant="primary" size="lg">
+      Large button
+    </Button>{' '}
+    <Button variant="outline-primary" size="lg">
+      Large button
+    </Button>
+  </div>
+  <div>
+    <Button variant="primary" size="sm">
+      Small button
+    </Button>{' '}
+    <Button variant="outline-primary" size="sm">
+      Small button
+    </Button>
+  </div>
+</>
+```
+
+##### Block Buttons
+
+```jsx live
+<>
+  <Button variant="primary" size="lg" block>
+    Block level button
+  </Button>
+  <Button variant="secondary" size="lg" block>
+    Block level button
+  </Button>
+</>
+```
+
+***
+
+## Button/Deprecated
+
 ##### Basic Usage
 
 ```jsx live=true
-<Button className="btn-primary">Hello World!</Button>
+<Button.Deprecated className="btn-primary">Hello World!</Button.Deprecated>
```

Comment:
Ah, I see the doc for the Deprecated one here - yeah, this part should stay at least.